https.go

```diff
@@ -533,10 +545,18 @@ func (proxy *ProxyHttpServer) NewConnectDialToProxy(httpsProxy string) func(netw
 }
 
 func (proxy *ProxyHttpServer) NewConnectDialToProxyWithHandler(
-	httpsProxy string,
-	connectReqHandler func(req *http.Request),
+  https_proxy string,
+  connectReqHandler func(req *http.Request) error,
+) func(network, addr string) (net.Conn, error) {
+	return proxy.NewConnectDialToProxyWithMoreHandlers(https_proxy, nil, nil)
+}
+
+func (proxy *ProxyHttpServer) NewConnectDialToProxyWithMoreHandlers(
+  https_proxy string,
```

Comment:
Please avoid snake case for the input variable names, keep it consistent with the codebase

Reply:
Fixed.